### PR TITLE
Fix internal links

### PR DESCRIPTION
**Pull request by typical000**
The problem was here:
https://github.com/cssinjs/cssinjs/commit/aa7caa647d52fa59dc71ea543e2f2ec7f1735c5e
Because `page` have no `org` property, in any case. Its taken from `pages.json`

### CODE REVIEW

File: src/utils/navigation.js [hunk 1]
```diff
@@ -63,7 +63,6 @@ export const findPage = (url) => {
     if (host === githubHost) {
       const meta = pathToMeta(pathname)
       if (
-          meta.org === page.org &&
```
Comment (kof) [inline src/utils/navigation.js]: hmm, so the org is allowed to be different? You sure its the right fix? Maybe meta.org or page.org parsing is wrong?

Comment (kof) [reply src/utils/navigation.js]: just wondering how this broke

Comment (typical000) [reply src/utils/navigation.js]: Ah, you are right. There is problem in detecting `defaultOrg`. Don't know how I missed this function :)
I'll try to debug more today

**Comment by typical000** [comment]
Another try :) https://github.com/cssinjs/cssinjs/pull/18/commits/d92cb7c415c68d9956c657702bca442adddaf9c6

Problem was here - https://github.com/cssinjs/cssinjs/commit/af38f64989b574a5be613e418439997b8c9e16ba#diff-70331ac9385f07bae06d3eda5886b3e5L15 just a wrong fix for eslint that throws `no-multi-assign`

**Comment by kof** [comment]
ltgm

**Pull request merged**